addPlant.php

```diff
@@ -0,0 +1,137 @@
+<?php
+            require_once 'src/PlantModel.php';
+
+            if (isset($_POST['newPlantName'])&&
+                isset($_POST['newPlantScientificName'])&&
+                isset($_POST['newPlantFamily'])&&
+                isset($_POST['newPlantImage'])&&
+                isset($_POST['newPlantDescription'])){
```

Comment:
The correct (as per the php coding standards) format for multi line if statements is like this:

```
if (
    condition &&
    condition &&
    condition
) {

}
```